### ISSUE TITLE
Fix react-headless

### ISSUE THREAD

**Issue by hamidra** [paritytech/polkadot-onboard#25]
After exporting the libs as module (type:module) the react-headless example fails since can't resolve the js as es modules.

```bash
ERROR in ../../packages/core/dist/index.js 1:0-39

Module not found: Error: Can't resolve './lib/walletAggregator' in '/Users/hra/Workspace/Parity/Git/wallet-sdk/packages/core/dist'
Did you mean 'walletAggregator.js'?
BREAKING CHANGE: The request './lib/walletAggregator' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

**Comment by SaltyCucumber** [comment]
Fixed the issue in https://github.com/paritytech/polkadot-onboard/pull/27
will need to update the versions and publish once approved

**Comment by hamidra** [comment]
fixed #28
published #29
